src/core/qgsmaplayerutils.cpp

```diff
@@ -166,3 +166,20 @@ QString QgsMapLayerUtils::launderLayerName( const QString &name )
 
   return laundered;
 }
+
+bool QgsMapLayerUtils::isOpenStreetMapLayer( QgsMapLayer *layer )
+{
+  if ( layer->providerType() == QStringLiteral( "wms" ) )
```

Comment:
Should this function not be a generic "layer hosted on osm server" as described in the docstring/implied by its name and not limited to `wms`?

Reply:
@uclaros , it won't be limited to  wms in the future, e.g. vector tile layers have been announced by OSMF, hopefully that'll come soon :crossed_fingers: 

That said, I'm also trying to avoid casting to broad a net so we don't erroneously flag non-tile content. You think the documentation should explicitly mention tiles?

Reply:
> so we don't erroneously flag non-tile content

Renaming the function to `isOpenStreetMapTileLayer()` then would also make sense, as well as mentioning it on the dosctring.